Input/input.css

```diff
@@ -0,0 +1,107 @@
+:root {
+  --gray: #3e4152;
+  --blue: #084298;
+  --red: #dc3545;
+  --text-color: #007fff;
+  --bg-code: #fafafa;
+}
+body {
+  font-family: "Poppins", sans-serif;
+  margin: 2rem;
+  color: var(--gray);
+}
+h1 {
+  font-size: 3rem;
+}
+
+h1,
+h2,
+p {
+  margin: 0.625rem 0;
+  line-height: 1.2;
+}
+
+h2 {
+  font-size: 2rem;
+}
+p {
+  font-size: 1.2rem;
+  padding: 0.8rem;
+}
+.input-box {
+  border: 1px solid var(--gray);
+  margin: auto;
+  width: 80%;
+  padding: 0.3rem 1rem;
+  border-radius: 6px;
+  padding-bottom:1rem;
+  text-align: center;
+}
+.fa {
+  margin-left: 1rem;
+  font-size: 1.5rem;
+  padding: 0.4rem;
+}
```

Comment:
you can give your class name for icons instead of using font awesome's inbuilt class for styling.